### DISCUSSION TITLE
Is there a way to prevent merging of parts

### DISCUSSION THREAD

**Discussion by Belstrekkie** [supermerill/SuperSlicer#2260] [Q&A] [answered]
Is there a way to prevent two objects that are placed upon each other to be merged during the slicing process in super/prusa slicer? I'm trying to print a nameplate in which I want to print a letter on top of a base plate. 
<img width="367" alt="example1" src="https://user-images.githubusercontent.com/34160776/150633458-2b832043-3090-4073-98ba-0f91c752dac0.png">
For best print quality, I do want the top layer of the base print printed monotonic with undisturbed lines. Both the base plate and the letter are separate objects. This works perfectly in Cura, in which both objects are sliced as separate objects, with a perfect top finish of the base plate: the left image is the top layer of the base plate (z=3.0) and the right images the first letter layer (z=3.2).
<img width="598" alt="Cura" src="https://user-images.githubusercontent.com/34160776/150633673-7f87014a-27a1-4eb8-9dcb-ab1ab9cfa22f.png">
But in super-/prusa slicer I'm not able to get a good finish of the base plate as the 2 objects are merged which complicates the top layer of the base plate: again, the left image is the top layer of the base plate (z=3.0) and the right images the first letter layer (z=3.2).
<img width="607" alt="Superslicer" src="https://user-images.githubusercontent.com/34160776/150633818-f44fb2b3-1ed1-40cc-b356-9c832a8f6193.png">
The only solution I found till now is to increase the Z-distance between both of objects, ignore the warnings during slicing and than manually change the Z-values for the letter in the produced Gcode. I hope there is a better one...

**Comment by neophyl** [comment, marked as answer]
Please post an example project (zipped up). There are ways to get an unbroken top infill like that and with an example I can modify it and post it back to show you the settings.

**Reply by Belstrekkie** [reply]
Below is an example of the above mentioned project.
[Example.zip](https://github.com/supermerill/SuperSlicer/files/7919943/Example.zip)

**Reply by neophyl** [reply]
PS/Susi will always merge parts when they are the same object.  You cant really stop that.  However for what it seems you want to do (having the unbroken layers printed below top features) then you can just adjust the Anchor solid Infill amount.  Basically set it to a suitable value in mm to force it to come under the detail on top.

You can also set that in a modifier so you can apply it to specific areas/features.  For such a simple shape as your example though its just easier to set it globally.  So I've still attached an example for you but in the end it only needed that single setting adjusting.
[Example_neo.zip](https://github.com/supermerill/SuperSlicer/files/7920764/Example_neo.zip)

Hope that helps and if I've not understood what you actually wanted please lmk.

**Reply by Belstrekkie** [reply]
Thanks! This solves the issue. I don't think I really understand the principle of this setting or why at had to be 5 mm in this example (I also tried other values now), but it works.

**Reply by supermerill** [reply]
It grows the top area by this value in all directions.
